src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveMinimalOrganisationsInfoTest.java

```diff
@@ -56,65 +58,58 @@
-                            bearerToken, HttpStatus.OK, IdamStatus.ACTIVE.toString(), true);
-
-            responseList.forEach(org -> orgResponseInfo.addAll(getOrganisationInfo(org)));
-
-            assertThat(orgResponseInfo)
-                    .contains(
-                            activeOrgs.get(0).getOrganisationIdentifier(),
-                            activeOrgs.get(0).getName(),
-                            activeOrgs.get(0).getContactInformation().get(0).getAddressLine1(),
-                            activeOrgs.get(1).getOrganisationIdentifier(),
-                            activeOrgs.get(1).getName(),
-                            activeOrgs.get(1).getContactInformation().get(0).getAddressLine1(),
-                            noAddressOrgs.get(0).getOrganisationIdentifier(),
-                            noAddressOrgs.get(0).getName(),
-                            noAddressOrgs.get(1).getOrganisationIdentifier(),
-                            noAddressOrgs.get(1).getName())
-                    .doesNotContain(
-                            pendingOrgs.get(0).getOrganisationIdentifier(),
-                            pendingOrgs.get(0).getName(),
-                            pendingOrgs.get(1).getOrganisationIdentifier(),
-                            pendingOrgs.get(1).getName());
-        }
+        setUpTestData();
+        List<OrganisationMinimalInfoResponse> responseList = (List<OrganisationMinimalInfoResponse>)
+            professionalApiClient.retrieveAllActiveOrganisationsWithMinimalInfo(
+                bearerToken, HttpStatus.OK, IdamStatus.ACTIVE.toString(), true);
+
+        responseList.forEach(org -> orgResponseInfo.addAll(getOrganisationInfo(org)));
+
+        assertThat(orgResponseInfo)
+            .contains(
+                activeOrgs.get(0).getOrganisationIdentifier(),
+                activeOrgs.get(0).getName(),
+                activeOrgs.get(0).getContactInformation().get(0).getAddressLine1(),
+                activeOrgs.get(1).getOrganisationIdentifier(),
+                activeOrgs.get(1).getName(),
+                activeOrgs.get(1).getContactInformation().get(0).getAddressLine1(),
+                noAddressOrgs.get(0).getOrganisationIdentifier(),
+                noAddressOrgs.get(0).getName(),
+                noAddressOrgs.get(1).getOrganisationIdentifier(),
+                noAddressOrgs.get(1).getName())
+            .doesNotContain(
+                pendingOrgs.get(0).getOrganisationIdentifier(),
+                pendingOrgs.get(0).getName(),
+                pendingOrgs.get(1).getOrganisationIdentifier(),
+                pendingOrgs.get(1).getName());
     }
 
     @Test
-    @SuppressWarnings("unchecked")
+    @ToggleEnable(mapKey = mapKey, withFeature = false)
     public void should_retrieve_403_when_API_toggled_off() {
-        if (!activeOrgsExternalEnabled) {
-            setUpTestData();
+        setUpTestData();
 
```

Comment:
this should also return 405

Reply:
what should we except here.....this should return HTTP status code - 405 as this is for DELETE
in other case, we should return 406. This is search api :) 405 or 406

Reply:
same above As discussed keeping 403 as is